picard/ui/savewarningdialog.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+#
+# Picard, the next-generation MusicBrainz tagger
+#
+# Copyright (C) 2023 Bob Swift
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+
+
+from PyQt5 import QtWidgets
+
+from picard.config import get_config
+
+
+class SaveWarningDialog():
+
+    def __init__(self):
+
+        actions = []
+        config = get_config()
+
+        if not config.setting['dont_write_tags']:
+            actions.append(_("overwrite existing metadata in the files"))
+        if config.setting['rename_files']:
+            actions.append(_("rename the files"))
+        if config.setting['move_files']:
+            actions.append(_("move the files"))
+
+        if actions:
+            header = _("This action will:")
+            footer = _("<strong>This action cannot be undone.</strong>  Do you want to continue?")
+            list_of_actions = ''
+            for action in actions:
+                list_of_actions += f'<li>{action}</li>'
```

Comment:
Actually those pseudo HTML parts also need to be translated even though they don't contain textual strings(I think it was the case in my example).
Mainly because of `dir` attribute that can be set by a translator for languages not using default left to right direction.
Same goes for HTML below.

See https://doc.qt.io/qt-6/richtext-html-subset.html#block-attributes